src/components/LetterLoad.vue

```diff
@@ -63,7 +70,9 @@ export default {
   },
   data () {
     return {
-      isSubmitted: true
+      date: new Date().toLocaleString(),
```

Comment:
I think we should perhaps pass `en-US` to ensure that the date is always formatted the same way. `new Date.toLocaleString('en-US')`

or alternatively, `new Intl.DateTimeFormat('en-US').format(new Date()))`

Reply:
Hello Glenn, I have updated the date format using:
```new Intl.DateTimeFormat('en-US').format(new Date())```